### DISCUSSION TITLE
Solocraft installation and/or settings

### DISCUSSION THREAD

**Discussion by Yoshi-01** [azerothcore/azerothcore-wotlk#19645] [Modules]
Hi, i've installed solocraft successfully. But when I enter a Ini (usualy with solo LFG) I get a Ingame Messige with placeholders and I cant see the icon of my pet (worlock). I add pics. Did I something wrong?
![2024-08-16_11h18_13](https://github.com/user-attachments/assets/b42dc31e-408b-4eca-8a40-5b68d3a5d6cc)
![2024-08-16_11h18_36](https://github.com/user-attachments/assets/486782a6-72e0-4edb-bf60-d8c917c3abd9)

**Comment by Nyeriah** [comment]
Please open an issue on the module repository and/or refer to the module installation.
